src/main/java/by/exadel/internship/service/FileService.java

```diff
@@ -0,0 +1,8 @@
+package by.exadel.internship.service;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.multipart.MultipartFile;
+
+public interface FileService {
+    String upload(MultipartFile multipartFile);
```

Comment:
MultipartFile is a controller level object/ Please pass byte array to service

Reply:
Done